http/auth.go

```diff
@@ -54,10 +55,38 @@ func (c *AuthenticationClient) Authenticated() bool {
 	return c.authenticated
 }
 
-func (c *AuthenticationClient) SetHeaders(headers http.Header) {
+func SetCookies(headers *http.Header, c *AuthenticationClient) {
+	var b bytes.Buffer
+	for _, cookie := range Cookies(c) {
+		b.WriteString(cookie.String())
+		b.WriteString("; ")
+	}
+	headers.Set("Cookie", b.String())
+}
+
+func Cookies(c *AuthenticationClient) []*http.Cookie {
+	cookies := make([]*http.Cookie, 0, 0)
```

Comment:
var cookies []*http.Cookie will to the same

Reply:
ok